docs/cli_usage.md

```diff
@@ -37,6 +37,15 @@ options:
   --hf-revision HF_REVISION
         A tag name, a branch name, or commit hash for the task HF dataset.
 
+  --wandb-project WANDB_PROJECT
+      The name of the Weights & Biases project to log runs to.
```

Comment:
Maybe: "Evaluation will be recorded to W&B only if `wandb-project` is given and `WANDB_API_KEY` environment variable is set".